ui/src/common/components/UnclickableTag/UnclickableTag.jsx

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Tag } from 'antd';
+import classnames from 'classnames';
+
+import './UnclickableTag.scss';
+
+class UnclickableTag extends Component {
+  render() {
+    const { children, className, ...otherProps } = this.props;
```

Comment:
you can set `children` like `...otherProps` too.

Reply:
do you mean ``otherProps.children``?